> The `py-multihash` requires python 3. But are you sure this requires python 3?

Hi @CMCDragonkai ,

There are test failures in `python2.7`.
It needs `zip_longest` which is not available in `python2.7`
So just to be on safer side, i had disabled this for `python2.7`

```
============================= test session starts ==============================
platform linux2 -- Python 2.7.17, pytest-4.6.8, py-1.8.1, pluggy-0.13.1
rootdir: /build/py-multibase-1.0.1
collected 0 items / 1 errors                                                   

==================================== ERRORS ====================================
___________________ ERROR collecting tests/test_multibase.py ___________________
ImportError while importing test module '/build/py-multibase-1.0.1/tests/test_multibase.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
tests/test_multibase.py:9: in <module>
    from multibase import encode, decode, is_encoded
multibase/__init__.py:9: in <module>
    from .multibase import encode, decode, get_codec, is_encoded, Encoding, ENCODINGS  # noqa: F401
multibase/multibase.py:4: in <module>
    from .converters import BaseStringConverter, Base16StringConverter, IdentityConverter, Base64StringConverter, \
multibase/converters.py:3: in <module>
    from itertools import zip_longest
E   ImportError: cannot import name zip_longest
!!!!!!!!!!!!!!!!!!! Interrupted: 1 errors during collection !!!!!!!!!!!!!!!!!!!!
=========================== 1 error in 0.10 seconds ============================
builder for '/nix/store/a8ic252x4d4s113gm04afqj52mi2dkqi-python2.7-py-multibase-1.0.1.drv' failed with exit code 2
error: build of '/nix/store/a8ic252x4d4s113gm04afqj52mi2dkqi-python2.7-py-multibase-1.0.1.drv' failed
[100] rakeshg@formbay-ml-2-C>                                                                                 
```

--- a/pkgs/development/python-modules/py-multibase/default.nix
+++ b/pkgs/development/python-modules/py-multibase/default.nix
@@ -1,0 +1,46 @@
+{ buildPythonPackage
+, fetchPypi
+, isPy27
+, lib
+, morphys
+, pytest
+, pytestrunner
+, python-baseconv
+, six
+}:
+buildPythonPackage rec {
+  pname = "py-multibase";
+  version = "1.0.1";
+  disabled = isPy27;
+
+  src = fetchPypi {
+    inherit pname version ;
+    sha256 = "6ed706ea321b487ba82e4172a9c82d61dacd675c865f576a937a94bca1a23443";
+  };
+
+  postPatch = ''
+    substituteInPlace setup.cfg --replace "[pytest]" ""
+    substituteInPlace setup.cfg --replace "python_classes = *TestCase" ""
+  '';
+
+  nativeBuildInputs = [
+    pytestrunner
+  ];
+
+  propagatedBuildInputs = [
+    morphys
+    six
+    python-baseconv
+  ];
+
+  checkInputs = [
+    pytest
+  ];
+
+  meta = with lib; {
+    description = "Multibase is a protocol for distinguishing base encodings and other simple string encodings";
+    homepage = "https://github.com/multiformats/py-multibase";
+    license = licenses.mit;
+    maintainers = with maintainers; [ rakesh4g ];
+  };
+}

--- a/pkgs/top-level/python-packages.nix
+++ b/pkgs/top-level/python-packages.nix
@@ -1129,6 +1129,8 @@ in {
 
   py-lru-cache = callPackage ../development/python-modules/py-lru-cache { };
 
+  py-multibase = callPackage ../development/python-modules/py-multibase { };
+
   py-multihash = callPackage ../development/python-modules/py-multihash { };
 
   py-radix = callPackage ../development/python-modules/py-radix { };

